projects/tomlkit/build.sh

```diff
@@ -15,6 +15,7 @@
 #
 ################################################################################
 pip3 install .
+pip3 install dictgen
```

Comment:
Could you move this to the Dockerfile?

Reply:
Have moved this to the Dockerfile and tested locally running using introspector. All working.